src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -95,6 +97,37 @@ protected override bool UnderAsyncAnonymousMethod(SyntaxToken token, SyntaxToken
                    current.GetLastToken().Span.End <= lastToken.Span.End;
         }
 
+        protected sealed override bool IsConfigureAwaitFalse(SyntaxNode node)
+        {
+            if (node is InvocationExpressionSyntax
+            {
+                Expression: MemberAccessExpressionSyntax
+                {
+                    Name:
+                    {
+                        Identifier:
+                        {
+                            ValueText: "ConfigureAwait"
+                        }
+                    }
+                },
+                ArgumentList:
+                {
+                    Arguments:
+                    {
+                        Count: 1
+                    } arguments
+                }
+            })
+            {
+                if (arguments[0].Expression is LiteralExpressionSyntax literalExpression
+                    && literalExpression.Kind() == SyntaxKind.FalseLiteralExpression)
+                    return true;
+            }
+
+            return false;
+        }
```

Comment:
this code is C# specific.  However, with SyntaxFacts, it feels like we could share this between C# and VB.

Reply:
the latter